crates/state/src/asm_state.rs

```diff
@@ -31,3 +33,33 @@ impl AsmState {
         &self.state
     }
 }
+
+/// Serde serialization of logs as hex strings.
+mod asm_log_entry_vec_hex {
```

Comment:
Is it not enough to add some attrs to `AsmLogEntry`?

Reply:
yeah, added.